addons/binding/org.openhab.binding.jeelink/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,224 @@
+                <label>@text/parameter.initcommands.label</label>
+                <description>@text/parameter.initcommands.description</description>
+            </parameter>
+		</config-description>
+
+	</bridge-type>
+
+	<!-- Lacrosse Temperature Sensor Thing Type -->
+	<thing-type id="lacrosse">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="jeelink" />
+		</supported-bridge-type-refs>
+
+        <label>@text/thing-type.lacrosse.label</label>
+        <description>@text/thing-type.lacrosse.description</description>
+
+		<channels>
+			<channel id="sensorId" typeId="sensor-id-channel" />
+			<channel id="sensorType" typeId="sensor-type-channel" />
+			<channel id="temperature" typeId="temperature-channel" />
+			<channel id="humidity" typeId="humidity-channel" />
+			<channel id="batteryNew" typeId="battery-new-channel" />
+			<channel id="batteryLow" typeId="battery-low-channel" />
+		</channels>
+
+		<config-description>
+			<parameter name="sensorId" type="text" required="true">
+				<label>@text/parameter.sensorid.label</label>
+				<description>@text/parameter.sensorid.description</description>
+			</parameter>
+			<parameter name="updateInterval" type="integer" required="true" min="5" max="3600" unit="s" step="5">
+                <label>@text/parameter.updateinterval.label</label>
+                <description>@text/parameter.updateinterval.description</description>
+				<default>60</default>
+			</parameter>
+			<parameter name="sensorTimeout" type="integer" required="true" min="5" max="3600" unit="s" step="5">
+                <label>@text/parameter.sensortimeout.label</label>
+                <description>@text/parameter.sensortimeout.description</description>
+				<default>60</default>
+			</parameter>
+			<parameter name="bufferSize" type="integer" required="true" min="0" max="100">
+                <label>@text/parameter.buffersize.label</label>
+                <description>@text/parameter.buffersize.description</description>
+				<default>20</default>
+			</parameter>
+			<parameter name="minTemp" type="decimal" required="true">
+                <label>@text/parameter.mintemp.label</label>
+                <description>@text/parameter.mintemp.description</description>
+				<default>19</default>
+			</parameter>
+			<parameter name="maxTemp" type="decimal" required="true">
+                <label>@text/parameter.maxtemp.label</label>
+                <description>@text/parameter.maxtemp.description</description>
+				<default>25</default>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<!-- EC3000 Power Monitor Thing Type -->
+	<thing-type id="ec3k">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="jeelink" />
+		</supported-bridge-type-refs>
+
+        <label>@text/thing-type.ec3k.label</label>
+        <description>@text/thing-type.ec3k.description</description>
+
+		<channels>
+			<channel id="sensorId" typeId="sensor-id-channel" />
```

Comment:
It changes if the battery in the sensor is changed. Since I am not really that knowledgeable with respect to openhab concepts and api, I had added a channel with a dedicated command to set the id. I will remove this completely, as now the id can be changed in the properties.